first_run.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+echo "Installing the required libraries for the 're-id-object-tracking' module..."
+
+# Function to check if libcudnn.so.9 is present
+check_cudnn() {
+    echo "Checking if libcudnn.so.9 is installed..."
+    if ldconfig -p | grep -q 'libcudnn.so.9'; then
+        echo "libcudnn.so.9 is already installed."
+        return 0
+    else
+        echo "libcudnn.so.9 is not installed."
+        return 1
+    fi
+}
+
+# Function to install cuDNN
+install_cudnn() {
+    echo "Starting the installation of cuDNN..."
+
+    # Download the cuDNN package
+    echo "Downloading cuDNN package..."
+    wget https://developer.download.nvidia.com/compute/cudnn/9.3.0/local_installers/cudnn-local-tegra-repo-ubuntu2204-9.3.0_1.0-1_arm64.deb
```

Comment:
`wget` might not be installed by default. Do we need an `apt-get install -y wget` in here?

Reply:
I don't known, I haven't got any trouble on fresh orin yet

Reply:
:shrug: Okay, leave as-is I guess